### ISSUE TITLE
Standardize latest/all behaviour for members/member-of endpoints

### ISSUE THREAD

**Issue by alexdunnjpl** [NASA-PDS/registry-api#399]
## 💡 Description
Post #396, /all and /latest endpoints still exist, but have no effect. Either restore their functionality, or remove them.

@jordanpadams I suggest we remove the endpoints, and have the members/member-of endpoints behave in the following manner:

- `products/{lid}/members` return the first-level members of the latest `lidvid` for `lid`
- `products/{lid}/members/members` return the second-level members of the latest `lidvid` for `lid`
- `products/{lid}/member-of` get the document for the latest lidvid for `lid`, and  return every lidvid specified in the appropriate ancestry attribute (should only ever be populated with lidvids afaik)
- `products/{lid}/member-of/member-of` get the document for the latest lidvid for `lid`, and  return every lidvid specified in the appropriate ancestry attribute (should only ever be populated with lidvids afaik)

**Comment by tloubrieu-jpl** [comment]
@alexdunnjpl @jordanpadams I agree with that proposal, the most simple the better since I don't feel like supporting lid only there is important except for the fact that they are sub-path of product/{lid} which we support (as Alex just told me !)

**Comment by jordanpadams** [comment]
@alexdunnjpl to clarify, do this still works?

```
/products/{identifier}/all
/products/{identifier}/latest
```

**Comment by jordanpadams** [comment]
@alexdunnjpl per our discussion at the stand-up, as long as those endpoints work, those are the only ones we care about. I think the others can go away because I don't really see the use cases there.

**Comment by alexdunnjpl** [comment]
@tloubrieu-jpl I'd tentatively suggest this as a candidate for my next deliverable
